libs/assets/signedasset.go

```
@@ -552,7 +552,7 @@ func buildSigKeys(store *DataSource, signers []string, currentTransfer *protobuf
 			if aggregatedPublicKey == nil {
 				aggregatedPublicKey = pubKey
 			} else {
-				_, aggregatedPublicKey = crypto.BLSAddG2(aggregatedPublicKey, pubKey)
+				aggregatedPublicKey, _ = crypto.BLSAddG2(aggregatedPublicKey, pubKey)
```

Might be worth adding an return error if the Pubkey fails to aggregate, although I know that it will result in the signature failing verification (which is covered), so it could be overkill. But it could help us get to the source of the verification failure more easily.

---

I shouldn't be ignoring errors, I'll add in an error handler.